apps/volontulo/views/offers.py

```diff
@@ -449,6 +449,16 @@ def post(request, slug, id_):  # pylint: disable=unused-argument
         if form.is_valid():
```

Comment:
but `request.user.is_authenticated()` and `request.user.is_active` is checked in nested if statements.